src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1958,6 +1958,27 @@ internal static bool CheckRefEscape(SyntaxNode node, BoundExpression expr, uint
             return false;
         }
 
+        internal static uint GetBroadestValEscape(BoundTupleExpression expr, uint scopeOfTheContainingExpression)
+        {
+            uint broadest = scopeOfTheContainingExpression;
+            foreach (var element in expr.Arguments)
+            {
+                uint valEscape;
+                if (element.Kind == BoundKind.TupleLiteral)
```

Comment:
>if [](start = 16, length = 2)

I would expect this test to be part of the switch in `GetValEscape`. It seems out of place here.

Reply:
Oh, I see, in GetValEscape we would be using Math.Max.

---
In reply to: [141214199](https://github.com/dotnet/roslyn/pull/22354#discussion_r141214199) [](ancestors = 141214199)